etc/submit_py3.sh

```diff
@@ -132,20 +132,17 @@ echo "======== WMAgent COMP Python bootstrap starting at $(TZ=GMT date) ========
 # slc6_ppc64le_gcc493 (from CMSSW): py2-future/0.18.2 python/2.7.15 py3-future/0.18.2 python3/3.8.2
 #
 # NOTE: all the ppc64le ScramArchs are actually pointing to: slc7_ppc64le_gcc820
+### UPDATE on 11 April, 2022: See a new map of CVMFS packages in:
+# https://github.com/dmwm/WMCore/pull/11077#issuecomment-1094814966
+
 # First, decide which COMP ScramArch to use based on the required OS and Architecture
 THIS_ARCH=`uname -m`  # if it's PowerPC, it returns `ppc64le`
-if [ "$THIS_ARCH" = "x86_64" ]
-then
-    THIS_ARCH="amd64"
-fi
-if [ "$REQUIRED_OS" = "rhel7" ]
-then
-    WMA_SCRAM_ARCH=slc7_${THIS_ARCH}_gcc630
-elif [ "$THIS_ARCH" = "amd64" ]
+# if this job can run at any OS, then use rhel7 as default
+if [ "$REQUIRED_OS" = "any" ]
 then
-    WMA_SCRAM_ARCH=slc6_${THIS_ARCH}_gcc700
+    WMA_SCRAM_ARCH=rhel7_${THIS_ARCH}
```

Comment:
I would rather define separate default flavor variable, e.g. `DEFAULT_OS=rhl7` somewhere on the top of the script and then use it here. This makes it easy to maintain and avoid changing defaults somewhere down in a code. I usually prefer to have common definitions on top of the script and then have a line saying `DO NOT EDIT below this line` to make clear separation between definitions and logic itself.

Reply:
That's a good suggestion, thanks Valentin.